plugins/admission/pkg/admission/validate.go

```diff
@@ -116,10 +118,23 @@ func (v *Validator) handleInternal(ctx context.Context, req admission.Request) (
 		logrus.Errorf("Error unmarshaling JSON")
 		return false, nil, nil, err
 	}
-	ownerReferences, ok := decoded["metadata"].(map[string]any)["ownerReferences"].([]any)
-	if ok && len(ownerReferences) > 0 {
-		logrus.Infof("Object has an owner - skipping")
-		return true, nil, nil, nil
+	if ownerReferences, ok := decoded["metadata"].(map[string]any)["ownerReferences"].([]any); ok && len(ownerReferences) > 0 {
+		ownerReference := ownerReferences[0].(map[string]any)
```

Comment:
shouldn't all owner references be checked for?

Reply:
Probably there is only one, but this may be a good idea to test all in the case someone tries to bypass the validation